packages/recommend-react/src/useFrequentlyBoughtTogether.ts

```diff
@@ -21,14 +22,26 @@ export function useFrequentlyBoughtTogether<TObject>({
   recommendClient,
   threshold,
   transformItems: userTransformItems,
-}: UseFrequentlyBoughtTogetherProps<TObject>) {
-  const [result, setResult] = useState<GetRecommendationsResult<TObject>>({
-    recommendations: [],
-  });
-  const { status, setStatus } = useStatus('loading');
+  initialResults: userInitialResults,
```

Comment:
does this still need to be renamed, it could be this right (with the usage changed too)?

```suggestion
  initialResults,
```